common/src/jni/main/cpp/conscrypt/native_crypto.cc

```diff
@@ -5676,50 +5696,6 @@ static inline bool decimal_to_integer(const char* data, size_t len, int* out) {
     return true;
```

Comment:
Think you can remove this helper too. Surprised the compiler didn't shout at you.

Reply:
It did, but it turns out we were compiling without -Werror and Android Studio doesn't surface C compiler warnings noticeably enough...

Fixed.  Doesn't really belong in this CL but it's not going to mess up the git blame.